recipes-kernel/linux/files/bcm2712-raspberrypi5-domd.dts

```diff
@@ -63,6 +63,59 @@
 				      0x00 0x9e 0x04 0x00 0x9f 0x04 0x00 0xa0 0x04>;
 		};
 
+		mip1: msi-controller@131000 {
+			compatible = "brcm,bcm2712-mip-intc";
+			reg = <0x10 0x00131000  0x0 0xc0>;
+			msi-controller;
+			interrupt-controller;
+			#interrupt-cells = <2>;
+			brcm,msi-base-spi = <247>;
+			/* Actually 20 total, but the others are
+			 * both sparse and non-consecutive */
+			brcm,msi-num-spis = <8>;
+			brcm,msi-offset = <8>;
+			brcm,msi-pci-addr = <0xff 0xffffe000>;
+			xen,path = "/axi/msi-controller@131000";
+			xen,force-assign-without-iommu;
+			xen,reg = <0x10 0x00131000 0x0 0x1000 0x10 0x00131000>;
+		};
+
+		pciex1: pcie@1000110000 {
+			compatible = "brcm,bcm2712-pcie";
+			device_type = "pci";
+			max-link-speed = <2>;
+			#address-cells = <3>;
+			#size-cells = <2>;
+			#interrupt-cells = <1>;
+			reg = <0x10 0x00110000  0x0 0x9310>;
+			interrupt-parent = <&gic>;
+			interrupts = <0x00 0xdf 0x04 0x00 0xe0 0x04>;
+			interrupt-names = "pcie", "msi";
+			interrupt-map-mask = <0x00 0x00 0x00 0x07>;
+			interrupt-map = <0x00 0x00 0x00 0x01 &gic 0x00 0xdb 0x04>,
+					<0x00 0x00 0x00 0x02 &gic 0x00 0xdc 0x04>,
+					<0x00 0x00 0x00 0x03 &gic 0x00 0xdd 0x04>,
+					<0x00 0x00 0x00 0x04 &gic 0x00 0xde 0x04>;
+			msi-controller;
+//			msi-parent = <&mip1>;
+			msi-parent = <&pciex1>;
```

Comment:
Please add the same HACK description as in below pciex node.

Reply:
Added

Reply:
Looks like you forgot to push your changes...

Reply:
Correct. I forget to push changes.
Fixed